scripts/init.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+packageName=${1:-}
+
+ROOT_PACKAGE_NAME=$(jq -r ".name" < package.json)
+if [ "$ROOT_PACKAGE_NAME" != "lisk-elements-monorepo" ]; then
+	echo "Please use the command in the project root directory."
+	exit 1;
+fi
+
+if [ -z "$packageName" ] || [[ "$packageName" =~ [^a-zA-Z0-9-] ]]; then
+	echo "Usage: npm run init -- PACKAGE_NAME"
+	exit 1
+fi
+
+packageDir="./packages/$packageName"
+# Just in case package folder doesn't exist yet.
+mkdir -p "$packageDir"
+
+files=(
+	".prettierrc.json"
+	".prettierignore"
+	".npmrc"
+	".nycrc"
+	".npmignore"
+	".babelrc"
+	".eslintignore"
+	"cypress.json"
+	"cypress"
+	"browsertest"
+	"scripts"
+)
+
+for i in "${files[@]}"
+do
+	if [ ! -e "$packageDir/${i}" ];	then
+		ln -vs "../../templates/$i.tmpl" "$packageDir/$i"
```

Comment:
We should use the long form of these options for intelligibility.

Reply:
That causes the error below:

```

Reply:
?